### ISSUE TITLE
Errors with step 3 of installation

### ISSUE THREAD

**Issue by Pacoup** [wet-boew/wet-boew-drupal#210]
While trying to install the Drupal WET, I have encountered the following errors in step 3 of https://github.com/wet-boew/wet-boew-drupal:

[root@li520-73 ~]# drush --version

```
drush version 5.6
```

[root@li520-73 ~]# drush make --prepare-install --no-gitinfofile --working-copy https://github.com/wet-boew/wet-boew-drupal/raw/master/distro.make drupal_wet

```
drupal-7.15 downloaded.                                              [ok]

Unable to patch drupal with                                          [error]
drupal-image_instance_checking-1559696-2.patch.

Unable to patch drupal with                                          [error]
1400256-field_info_collate_fields-7.patch.

Unable to clone wetkit from                                          [error]
https://github.com/wet-boew/wet-boew-drupal.git.
```

---

I had the same error on Ubuntu, this is my CentOS build, and I installed Drush Make and it resolved the issue. However, according to this page (http://drupal.org/project/drush_make/), Drush Make is included in version 5 onwards, and I have the latest 5.6.

Anything I'm doing wrong?

**Comment by sylus** [comment]
What is your version of git? can you paste the drush --debug output.

**Comment by Pacoup** [comment]
Forgot to install Git.
But now when I try to install it, the folder I specify never gets generated.

Here's the debug log: http://pacoup.com/download/debuglog.txt

**Comment by sylus** [comment]
It seems there is an error during drush make in applying the patch: 

Unable to patch rubik with rubik-table_input_fluid-944534-5.patch
Unable to patch workbench_moderation with workbench_moderation-is_current-1260112-15.patch.

Yet looking at my travis build (http://travis-ci.org/#!/wet-boew/wet-boew-drupal) I am able to patch them. What is your version of git?

**Comment by Pacoup** [comment]
1.7.1

**Comment by Pacoup** [comment]
I installed drush like this:

<pre>pear channel-discover pear.drush.org
pear install drush/drush</pre>


As is outlined here http://drupal.org/project/drush/

Running CentOS 6. Everything's updated.

**Comment by sylus** [comment]
Your git is out of date can you try to update to 1.7.11? That should fix everything, earlier versions of git + drush make have had problems applying certain patches.

**Comment by Pacoup** [comment]
I did update Git, but got the same error. Turns out the culprit was the missing patch utility.

<pre>sh: patch: command not found [12.98 sec, 7.87 MB]</pre>


It's all ok now.

Maybe it would be worthwhile to add to the wiki a list of required packages for the installation, such as bzr and patch (and I think you eventually need php-gd for Drupal itself, at least the web-based installer had asked for it when I tried to install it that way).
